crates/xilem_masonry/src/view/label.rs

```diff
@@ -0,0 +1,92 @@
+use masonry::{widget::WidgetMut, ArcStr, WidgetPod};
+
+use crate::{ChangeFlags, Color, MasonryView, MessageResult, TextAlignment, ViewCx, ViewId};
+
+pub fn label(label: impl Into<ArcStr>) -> Label {
+    Label {
+        label: label.into(),
+        text_color: Color::default(),
```

Comment:
Hmm, `Color` implementing `Default` is weird - the default would surely depend on the use case?

Reply:
I can instead make it black (which seems to be the default) to be more clear. I agree, looking at it more closely, because default for the alpha value seems to be 0 (which I don't think is a good default). Maybe remove `Default` for `peniko::Color` alltogether (or make it explicitely black with opaque alpha)?

Reply:
Yeah, removing that `Default` is worth a PR, at worst to get a reason it should stay

Reply:
I've made it `Color::BLACK` for now in this PR.

Reply:
if we kept `Default`, this could just use functional record update syntax,
since `label` is the only non-default field.  Agree about the alpha value.
 
```
Label {
  label: label.into(),
  ..Default::default()
}
```

Reply:
I'm not sure I follow. `Label: Default` would not be a good API, because there's nearly no use for an empty label.

(Plus, I think that would allocate without need, which isn't great.)

Reply:
Of course, I just momentarily forgot functional record update syntax requires the structure itself to be `Default` not just the fields.